README.md

```diff
@@ -39,7 +39,7 @@ Tests are written using the [official test](https://github.com/ash-shell/test) m
 You can run tests by running this command, after Yaml-Parse is installed:
 
 ```sh
-ash test yamlparse
+ash test yaml-parse
```

Comment:
Let's change this to `ash test github.com/ash-shell/yaml-parse`... We both have different aliases for this package, and it's possible other people will.. The full package name will prevent any issues!